deploy/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:bionic
+MAINTAINER soheil@gmail.com
+
+ENV CREATE_MYSQL_BASIC_USER_AND_DB true
+ENV MYSQL_USER_NAME dvwa
+ENV MYSQL_USER_DB dvwa
+ENV MYSQL_USER_PASS p@ssw0rd
```

Comment:
Might be interesting to move this into environment file

Reply:
Is the intent to allow users to change set their own password using environment variables? I put this here so it was obvious that additional user and database was being created for DVMA, if someone changes the php.config they know to change it here also, I could have hard-coded it in the create_mysql_users.sh. but instead made them configured at this top level.